### PR TITLE
ref(pii): Added hide caret logic

### PR DESCRIPTION
**Pull request by priscilawebdev**
**Problem:**

When scrolling through selections with the keyboard, the mousecursor still may point to somewhere else on the list. So the selected item and the hovered item are out of sync

**Solution:**

the cursor will be hidden if the user scrolls to a suggestion item. When the user clicks again, the cursor will be displayed

### CODE REVIEW

**Review by untitaker — APPROVED** [review]

File: src/sentry/static/sentry/less/forms.less [hunk 1]
```diff
@@ -154,12 +154,12 @@ label {
 }
 
 .has-error .form-control {
-  border-color: #c9c0d1;
+  border-color: #c9c0d1 !important;
```
Comment (billyvg) [inline src/sentry/static/sentry/less/forms.less:157]: Why are we adding this? this could have unwanted effects in other places

**Review by billyvg — COMMENTED** [review]
Could you add a gif of what this PR does?

File: src/sentry/static/sentry/less/forms.less (hunk 1 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/less/forms.less:157]: yes I know but at the moment if we don't add !important tag, some global selector is setting a red color that we don't use... it's wrong.

**Comment by priscilawebdev** [comment]
Kapture 2020-05-18 at 20 38 29

**Review by billyvg — APPROVED** [review]

**Pull request merged**